From 8b19ba798cbc29ef66127675dcff4e0a7a896087 Mon Sep 17 00:00:00 2001
From: Srujana Challa <schalla@marvell.com>
Date: Fri, 8 Jan 2021 19:32:17 +0530
Subject: [PATCH 1057/1921] octeontx2-af: Fix warnings from CPT with W=1 flag

This patch fixes warnings with W=1 flag form CPT.

Fixes: 5f64f22c60da ("octeontx2-af: Add new mailbox for CPT stats")

Signed-off-by: Srujana Challa <schalla@marvell.com>
Change-Id: Ic375665d446afa13555f86037744165d6b145f37
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/43156
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 drivers/net/ethernet/marvell/octeontx2/af/rvu_cpt.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/net/ethernet/marvell/octeontx2/af/rvu_cpt.c b/drivers/net/ethernet/marvell/octeontx2/af/rvu_cpt.c
index 0c6adea11d71..45829db511ce 100644
--- a/drivers/net/ethernet/marvell/octeontx2/af/rvu_cpt.c
+++ b/drivers/net/ethernet/marvell/octeontx2/af/rvu_cpt.c
@@ -546,7 +546,7 @@ int rvu_mbox_handler_cpt_rd_wr_register(struct rvu *rvu,
 	return CPT_AF_ERR_ACCESS_DENIED;
 }
 
-void get_ctx_pc(struct rvu *rvu, struct cpt_sts_rsp *rsp, int blkaddr)
+static void get_ctx_pc(struct rvu *rvu, struct cpt_sts_rsp *rsp, int blkaddr)
 {
 	rsp->ctx_mis_pc = rvu_read64(rvu, blkaddr, CPT_AF_CTX_MIS_PC);
 	rsp->ctx_hit_pc = rvu_read64(rvu, blkaddr, CPT_AF_CTX_HIT_PC);
@@ -575,7 +575,7 @@ void get_ctx_pc(struct rvu *rvu, struct cpt_sts_rsp *rsp, int blkaddr)
 	rsp->rxc_zombie_sts = rvu_read64(rvu, blkaddr, CPT_AF_RXC_ZOMBIE_STS);
 }
 
-void get_eng_sts(struct rvu *rvu, struct cpt_sts_rsp *rsp, int blkaddr)
+static void get_eng_sts(struct rvu *rvu, struct cpt_sts_rsp *rsp, int blkaddr)
 {
 	u16 max_ses, max_ies, max_aes;
 	u32 e_min = 0, e_max = 0;
-- 
2.31.1

